packages/@react-spectrum/table/test/TableDnd.test.js

```diff
@@ -2848,7 +2867,43 @@ describe('TableView', function () {
       expect(dragHandle.style.position).toBe('');
     });
 
-    it('should not display the drag handle on hover, press, or keyboard focus for disabled/non dragggable items', async function () {
+    it('should display the drag handle on hover, press, or keyboard focus for disabled/non dragggable items with disabledBehavior="select"', async function () {
+      function hasDragHandle(el) {
+        let buttons = within(el).queryAllByRole('button');
+        if (buttons.length === 0) {
+          return false;
+        }
+        return buttons[0].getAttribute('draggable');
+      }
+
+      let {getByRole} = render(
+        <DraggableTableView tableViewProps={{disabledBehavior: 'select', disabledKeys: ['a']}} />
+      );
+
+      let grid = getByRole('grid');
+      let rowgroups = within(grid).getAllByRole('rowgroup');
+      let rows = within(rowgroups[1]).getAllByRole('row');
+
+      await user.tab();
+      expect(hasDragHandle(rows[0])).toBeTruthy();
+      moveFocus('ArrowDown');
+      expect(hasDragHandle(rows[1])).toBeTruthy();
+
+      fireEvent.pointerDown(rows[0], {button: 0, pointerId: 1});
+      expect(hasDragHandle(rows[0])).toBeTruthy();
+      fireEvent.pointerUp(rows[0], {button: 0, pointerId: 1});
```

Comment:
if you wanted to do this the userEvent way, you could do 
```
pointer('[MouseLeft>]') // press the left mouse button
pointer('[/MouseLeft]') // release the left mouse button
```

Reply:
ah, good call, just copy pasta'd most of these but I'll go ahead and update them